helm/net-exporter/templates/daemonset.yaml

```diff
@@ -2,7 +2,7 @@ kind: DaemonSet
 apiVersion: apps/v1
 metadata:
   name: net-exporter
-  namespace: {{ .Values.exporter.namespace }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
In CP we need to install that in `monitoring`. Does that break anything?

Reply:
Let me quickly check on that.

Reply:
Checked on viking so should be fine IMO:

```
$ k get app -A -l app=net-exporter -o json | jq '.items[].spec.namespace'
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
"kube-system"
```

Reply:
```net-exporter                                       	12      	Mon May 11 16:33:41 2020	DEPLOYED	net-exporter-chart-1.0.0-07c040837b3a59911f29988dffdc72fc...	                                              	draughtsman```
It will install inside `draughtsman` namespace then. I think we need other way around.

Reply:
I'm moving it to collection based deployment.

Reply:
Deploying with draughtsman is already disabled.

Reply:
Yes this makes sense to me. The release namespace is set by chart-operator and flows from the namespace in the spec of the app CRs.

In tenant clusters we use kube-system. In the control plane we can override to use monitoring.

Reply:
Do you know how to set `monitoring` namespace in .spec.namespace? 
For example in here, https://github.com/giantswarm/aws-app-collection/blob/master/helm/aws-app-collection-chart/templates/kubernetesd-unique.yaml#L28 
It looks like it hard-coded to giantswarm always.

Reply:
Namespace got added as an optional parameter to push to app collection.

https://github.com/giantswarm/g8s-oauth2-proxy/blob/8b2305ea187556f1d7aea6fa840b43282d3c5da0/.circleci/config.yml#L43